packages/modules/link-modules/src/migration/index.ts

```diff
@@ -332,24 +332,30 @@ export class MigrationsExecutionPlanner implements ILinkMigrationsPlanner {
     }[] = []
 
     for (let trackedTable of trackedTables) {
-      const newTableName = this.#linksEntities.find((entity) => {
+      const linkEntity = this.#linksEntities.find((entity) => {
         return (
           entity.linkDescriptor.fromModel ===
             trackedTable.link_descriptor.fromModel &&
           entity.linkDescriptor.toModel ===
             trackedTable.link_descriptor.toModel &&
-          entity.linkDescriptor.fromModule ===
-            trackedTable.link_descriptor.fromModule &&
-          entity.linkDescriptor.toModule ===
-            trackedTable.link_descriptor.toModule
+          entity.linkDescriptor.fromModule.toLowerCase() ===
+            trackedTable.link_descriptor.fromModule.toLowerCase() &&
+          entity.linkDescriptor.toModule.toLowerCase() ===
+            trackedTable.link_descriptor.toModule.toLowerCase()
         )
```

Comment:
Do we want to do `toLowerCase` or convert the module name to `snake_case`? I am not 100% sure either